assets/src/dashboard/theme.js

```diff
@@ -31,6 +31,10 @@ export const GlobalStyle = createGlobalStyle`
 	*::before {
 		box-sizing: border-box;
     }
+    
+  h1, h2, h3, h4, h5, h6, p, a {
+    margin: 0;
```

Comment:
we're applying margin:0 to our text elements individually. Doing this here makes things more consistent and eliminates the excess line on each element.